doc/rm/python_coding_style.md

```diff
@@ -192,3 +193,8 @@ This will provide support for `--help` and `-h` to get usage information.
 Every command line program should provide `--version` to provide standard version information.
 This lists the git repositary information for the tool and the version numbers of any Python packages that are used.
 The `show_and_exit` routine in `reggen/version.py` can be used to do this.
+
+Options that consume an arbitrary number of command line arguments with `nargs="*"` or `nargs="+"` should be avoided wherever possible.
+Comma separated lists can be passed to form single arguments and can be split after arguments are parsed.
```

Comment:
That's ok. Another approach is to use the [`'append'` action](https://docs.python.org/3/library/argparse.html#action), and do things like `--foo x --foo y`. I'm not sure I have strong feelings about which approach is better, but this avoids problems where e.g. the argument to foo really wants a comma in it. Maybe mention both options?